plugins/view-resources/src/actionImpl.ts

```diff
@@ -107,6 +114,11 @@ contextStore.subscribe((it) => {
   $contextStore = it
 })
 
+let $selectionStore: SelectionStore
```

Comment:
You could use get method from 'svelte/store' package.

`import { get } from 'svelte/store'`

Reply:
But his is minor, could be done next time.

Reply:
Yeah ... Maybe we should get rid of this global variable